core/infra/protobuf/api.pb.cc

```diff
@@ -340,10 +340,11 @@ void protobuf_AssignDesc_api_2eproto() {
       GOOGLE_PROTOBUF_GENERATED_MESSAGE_FIELD_OFFSET(Transaction, _internal_metadata_),
       GOOGLE_PROTOBUF_GENERATED_MESSAGE_FIELD_OFFSET(Transaction, _is_default_instance_));
   ConsensusEvent_descriptor_ = file->message_type(14);
-  static const int ConsensusEvent_offsets_[3] = {
+  static const int ConsensusEvent_offsets_[4] = {
```

Comment:
1. generated code is unreadable
2. generated code can be moved to the `gen` dir, so during lurking that should be intuitive that something need to be generated here